pkg/storage/client_replica_test.go

```diff
@@ -1518,26 +1518,11 @@ func TestSystemZoneConfigs(t *testing.T) {
 	ctx := context.TODO()
 	defer tc.Stopper().Stop(ctx)
 
-	// Before moving forward, pre-split the keys for the system zone configs.
-	// If we don't do this, the test can flake on the splits happening at an
-	// inopportune time (or on the replicate queue blocking on a needed split
-	// if we disable the split queue).
-	splitKeys := []roachpb.Key{
-		keys.MakeTablePrefix(keys.MetaRangesID),
-		keys.MakeTablePrefix(keys.TimeseriesRangesID),
-		keys.MakeTablePrefix(keys.SystemRangesID),
-	}
-	for _, key := range splitKeys {
-		if _, _, err := tc.SplitRange(key); err != nil {
-			t.Fatalf("failed to split at key %s: %s", key, err)
-		}
-	}
-
```

Comment:
They are always presplit as a direct result of this checkin. System table IDs 16, 17 and 18 were reserved to hold custom zone config information for non-table ranges (Meta, time series, system), so the new table has to get ID 19. The logic in `ComputeSplitKey` assumes that table IDs are sequential, so now that it has table 19 it creates splits for 16, 17 and 18, even though they aren't actually table.

The real question I have, now that I think about it more, is why were these ranges being split before?  This test says "These ranges may get split at an inopportune time", but these are not real tables, so I don't think they ever should have been split at any point. @a-robinson Can you shed some light on that?

In the meantime, I've updated the commit message to provide more detail around this oddity.